ts-relayer-tests/src/ics721.spec.ts

```diff
@@ -186,6 +244,115 @@ test.serial("transfer NFT", async (t) => {
   t.is(osmoAddr, tokenOwner.owner);
 });
 
+test.serial("transfer NFT with incoming proxy: osmo -> wasmd", async (t) => {
```

Comment:
relayer test for incoming proxy with 2 scenarios:
1) nft send to WLed channel
2) nft send to unknown channel